docs/cri/config.md

```diff
@@ -232,6 +232,11 @@ version = 2
 
   # 'plugins."io.containerd.grpc.v1.cri".registry' contains config related to the registry
   [plugins."io.containerd.grpc.v1.cri".registry]
+    # Specifies a directory to look for registry configs in.
+    # Dir can be used just like /etc/docker/certs.d OR can contain a hosts.toml with more specific configurations.
+    #
+    # NOTE: Specifying this will cause the cri plugin to ignore any other registry configs specified in this configuration file.
```

Comment:
if we are going to override all of this registry section.. we should consider doing it at the containerd config level and deprecate the cri registry config

Reply:
What's the "right" way to do this deprecation?

TBH I tried to make both work but its pretty tricky and I thought likely not worth the effort.

Reply:
gulp... typically yes.